### PR TITLE
Add goat emoji

### PR DESCRIPTION
**Pull request by RobertAKARobin**
Bitbucket doesn't support the ⛏ emoji, so instead we use 🐐, as in, ["This gets my goat."](https://www.dictionary.com/browse/get-someone-s-goat)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Using CREG (Code Review Emoji Guide) puts more ownership on the reviewer to give
...
 | 📝  |   `:memo:`   | This is an explanatory note, fun fact, or relevant commentary that does not require any action.                                                                                     |
-|  ⛏  |   `:pick:`   | This is a nitpick. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
+|  ⛏🐐  |   `:pick:` `:goat:`   | This is a nitpick, or, this gets my goat<sup>[*](https://www.dictionary.com/browse/get-someone-s-goat)</sup>. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
 |  ♻️  | `:recycle:`  | Suggestion for refactoring. <br /><br /> Should include enough context to be actionable and not be considered a nitpick. |
 |  🏕  | `:camping:`  | Here is an opportunity, not directly related to your changes, for us to leave the campground [code] cleaner than we found it.                                                       |
```
Comment (cosydney) [inline README.md:23]: 👍

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Using CREG (Code Review Emoji Guide) puts more ownership on the reviewer to give
...
 | 🌱  | `:seedling:` | Planting a seed for future. <br /><br /> An observation or suggestion that is not a change request, but may have larger implications. Generally something to keep in mind for the future. |
 | 📝  |   `:memo:`   | This is an explanatory note, fun fact, or relevant commentary that does not require any action.                                                                                     |
-|  ⛏  |   `:pick:`   | This is a nitpick. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
+|  ⛏🐐  |   `:pick:` `:goat:`   | This is a nitpick, or, this gets my goat<sup>[*](https://www.dictionary.com/browse/get-someone-s-goat)</sup>. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
```
Comment (cosydney) [inline README.md:21]: 🤔

File: README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Using CREG (Code Review Emoji Guide) puts more ownership on the reviewer to give
 | 🤔💭 | `:thinking:` `:thought_balloon:` | Let me think out loud here for a minute. <br /><br /> I might express concern, suggest an alternative solution, or walk through the code in my own words to make sure I understand. |
 | 🌱  | `:seedling:` | Planting a seed for future. <br /><br /> An observation or suggestion that is not a change request, but may have larger implications. Generally something to keep in mind for the future. |
 | 📝  |   `:memo:`   | This is an explanatory note, fun fact, or relevant commentary that does not require any action.                                                                                     |
```
Comment (cosydney) [inline README.md:20]: ❓

**Comment by kopach** [comment]
I also would prefer to change ⛏️ emoji to something else, but for different reason. Because it's visually very similar to 🔧 
Another option is using Upside-down-face 🙃 emoji suggested in original post https://daniel.town/emoji-code-review/
It's more neutral but still very distinguishable (people are particularly good at recognizing faces)

---

> and as for emojis - I personally prefer using emoji characters like ⛏️ insaed of  `:pick:` - that works everywhere, I believe even in Bitbucket

**Comment by RobertAKARobin** [comment]
Bitbucket now does support ⛏️

**Pull request closed without merging**